rpc/common/moc_common_common.proto

```diff
@@ -385,7 +387,8 @@ enum HyperVGeneration {
 enum DiskFileFormat {
 	DiskFileFormatVHDX =  0;
 	DiskFileFormatVHD = 1;
-	DiskFileFormatUNKNOWN = 2;
+	DiskFileFormatVHDSet = 2;
```

Comment:
Why repurposing the existing enum? New ones should be appended.

Reply:
Fixed